helpers/notification_helper.py

```diff
@@ -145,6 +146,23 @@ def send_award_update(cls, event):
         notification.send(keys)
 
     @classmethod
+    def send_match_video(cls, match):
+        """
+        Sends match_video and event_match_video notifications
+        If the match is current, MatchVideoNotification is sent.
+        Otherwise, EventMatchVideoNotification is sent
+        """
+        match_users = set(PushHelper.get_users_subscribed_to_match(match, NotificationType.MATCH_VIDEO))
+        event_users = set(PushHelper.get_users_subscribed_to_event(match.event.get(), NotificationType.MATCH_VIDEO))
+        users = match_users.union(event_users)
```

Comment:
We'll also need to query one more thing. If we subscribe to a team's match videos, we'll need to find all those who are subscribed to the each of the match's team keys